experiment_list.py

```diff
@@ -193,11 +208,20 @@ def check(self):
             if expt.region is None:
                 print("Warning: Experiment %s has no region" % str(expt.expt_id))
 
-    def distance_plot(self, pre_type, post_type, plots=None, color=(100, 100, 255)):
+    def distance_plot(self, pre_type, post_type, calcium, age, plots=None, color=(100, 100, 255)):
         # get all connected and unconnected distances for pre->post
         probed = []
         connected = []
-        for expt in self:
+        if calcium is not None:
```

Comment:
This should go in the other script as well -- don't mix plotting and selection in one place